yt/frontends/cf_radial/io.py

```diff
@@ -0,0 +1,35 @@
+"""
+CF-Radial-specific IO functions
+
+
+
+"""
+
+import numpy as np
+
+from yt.utilities.io_handler import BaseIOHandler
+
+
+class CFRadialIOHandler(BaseIOHandler):
+    _particle_reader = False
+    _dataset_type = "cf_radial"
+
+    def _read_fluid_selection(self, chunks, selector, fields, size):
```

Comment:
I am genuinely pleased at how short this file is, given how complicated they often can be!